webapp/src/app.tsx

```diff
@@ -2874,6 +2874,16 @@ export class ProjectView
         tc.toggleHint(showFullText);
     }
 
+    ///////////////////////////////////////////////////////////
+    ////////////     User alert/notifications     /////////////
+    ///////////////////////////////////////////////////////////
+
+    pokeUserActivity() {
+        // animate tutorial hint after some time of user inactivity
+        let tc = this.refs["tutorialcard"] as tutorial.TutorialCard;
```

Comment:
i think we should check the state of the app, e.g. in the editor with tutorial open so that we can start poping other hints in the future.